tests/core/test_cost_calculation.py

```diff
@@ -64,7 +64,7 @@ async def test_basics(self, rust_checker: bool):
         )
         coinbase = None
         for coin in blocks[2].get_included_reward_coins():
-            if coin.puzzle_hash == ph:
+            if coin.puzzle_hash == ph and coin.amount == 250000000000:
```

Comment:
this change makes us pick the same coinbase coin every time we run the test. Otherwise it may sometimes pick the 1.75 coin, and sometimes pick the 0.25 coin. Without this, we get different programs and different costs in different runs of the test.